docs/reference-implementation/Troubleshooting.md

```diff
@@ -0,0 +1,82 @@
+# Troubleshooting guide
+
+It's inevitable in a system with the complexity of AlwaysOn that issues and errors occur. This living document maintains a list of solutions for common errors, which are not directly caused by the AlwaysOn code (i.e. are outside of control of the development team and cannot be fixed as bugs in the codebase).
+
+## Deployment issues
+
+### Infrastructure Deployment stages
+
+**Error:**
+```
+A resource with the ID "/subscriptions/[...]/diagnosticSettings/frontdoorladiagnostics" already exists - to be managed via Terraform this resource needs to be imported into the State.
+```
+
+
+**Description:** Occurs on global Terraform apply, specifically azurerm_monitor_diagnostic_setting.frontdoor, but it can happen on stamps too (various resources).
```

Comment:
```suggestion
**Description:** Occurs on global Terraform apply, specifically `azurerm_monitor_diagnostic_setting.frontdoor`, but it can happen on stamps too (various resources).
```